app/db/models/document/physical_document.py

```diff
@@ -0,0 +1,45 @@
+import sqlalchemy as sa
+from app.db.session import Base
+
+
+class PhysicalDocument(Base):
+    """A physical document.
+
+    Representation of a document that exists in the real world.
+    Owned and updated only by the pipeline.
+    """
+
+    __tablename__ = "physical_document"
+
+    id = sa.Column(sa.Text, primary_key=True)
+    title = sa.Column(sa.Text, nullable=False)
+    md5_sum = sa.Column(sa.Text, nullable=True)
+    source_url = sa.Column(sa.Text, nullable=True)
+    date = sa.Column(sa.DateTime, nullable=False)
+    format = sa.Column(sa.Text, nullable=True)
+
+
+class Language(Base):
+    """Note, moved from deprecated."""
+
+    __tablename__ = "language"
+
+    id = sa.Column(sa.SmallInteger, primary_key=True)
```

Comment:
are we trying to save memory? I'm very nervous of ever using small integers...

Reply:
... this was blindly copied from the deprecated module - we can migrate it I suppose?

Reply:
ugh, I knew I'd seen this before! Might be worth taking the time to migrate now...

Reply:
`int2` really scares me